ambassador/ambassador/config/config.py

```diff
@@ -74,6 +74,11 @@ def __init__(self, schema_dir_path: Optional[str]=None) -> None:
             # with our ACResource class.
             schema_dir_path = resource_filename(Requirement.parse("ambassador"), "schemas")
 
+        self.statsd = {'enabled': (os.environ.get('STATSD_ENABLED', '').lower() == 'true')}
+        if self.statsd['enabled']:
+            self.statsd['interval'] = os.environ.get('STATSD_FLUSH_INTERVAL', '1')
+            self.statsd['ip'] = socket.gethostbyname(os.environ.get('STATSD_HOST', 'statsd-sink'))
```

Comment:
What happens if `socket.gethostbyname` can't find the host?

Reply:
Yeah, everything crashes :fire: Thanks!
I've caught this exception, PTAL :)